addhost.php

```diff
@@ -181,6 +196,7 @@
     c_echo(
         "\n".$config['project_name_version'].' Add Host Tool
 
+    Usage (ICMP)     : ./addhost.php [-g <poller group>] [-f] -P <%Whostname%n> [os] [hardware]
```

Comment:
Move the example for this below SNMP so as to not cause confusion. Change `Usage (ICMP)` to `Usage (ICMP Only)`.